src/cli/base.ts

```diff
@@ -21,18 +22,22 @@ export interface LinterArg {
 	ui5Config?: string;
 }
 
-// yargs type defition is missing the "middelwares" property for the CommandModule type
+// yargs type definition is missing the "middlewares" property for the CommandModule type
 interface FixedCommandModule<T, U> extends CommandModule<T, U> {
 	middlewares: MiddlewareFunction<U>[];
 }
 
 const lintCommand: FixedCommandModule<object, LinterArg> = {
-	command: "$0",
+	command: "$0 [files...]",
 	describe: "Runs linter",
 	handler: handleLint,
 	middlewares: [baseMiddleware],
 	builder: function (args: Argv<object>): Argv<LinterArg> {
-		args.usage("Usage: $0 [options]")
+		args.usage("Usage: $0 [files...] [options]")
+			.positional("files", {
+				describe: "List of patterns to lint",
+				type: "string",
```

Comment:
I think this is missing `array: true` (see https://github.com/SAP/ui5-cli/blob/d33c3ab8a63345e199d731a0895ec8db9a32abde/lib/cli/commands/build.js#L42)

However, even then it doesn't seem to transform single positionals into an array. 

Executing `ui5lint --files webapp/controller/App.controller.js` in the troublesome app yields the following error:
```
Error Message:
patterns.map is not a function

Stack Trace:
TypeError: patterns.map is not a function
    at buildPatterns (/Users/d054485/SAPDevelop/dev/fnd/ui5-linter/src/linter/linter.ts:278:18)
    at resolveReader (/Users/d054485/SAPDevelop/dev/fnd/ui5-linter/src/linter/linter.ts:337:28)
```

Reply:
It should be `ui5lint webapp/controller/App.controller.js`

The `--files` option I suppose, comes from `yargs`. I have updated the code to enforce array for that.